src/main/java/frc/robot/Constants.java

```diff
@@ -91,16 +119,46 @@ public static final class ArmConstants{
         public static final double armMaxVel = 65;
         public static final double armMaxAccel = 85;
 
+
+        public static final double minArmShootAngle = 75;
+        public static final double maxArmShootAngle = 40;
         //FIXME: create lookup table
         public static final double [][] armLookupTable = {
-            {/* distance to target, arm angle */}
+            {1, 75},
+            {3, 65}
         };
 
+        public static double getArmAngleFromDistance(double distance) {
+
+            if(distance < armLookupTable[0][0]) {
+                return minArmShootAngle;
+            }
+            if(distance > armLookupTable[armLookupTable.length-1][0]) {
+                return maxArmShootAngle;
+            }
+
+            double[] smaller = new double[2];
+            double[] larger = new double[2];
+
+            for(int i = 0; i < armLookupTable.length-2; i++) {
+                if(distance >= armLookupTable[i][0] && distance <= armLookupTable[i+1][0]) {
+                    smaller = armLookupTable[i];
+                    larger = armLookupTable[i+1];
+                    break;
+                }
+            }
+            //Y = Y1 + (X - X1) * ((Y2 - Y1)/(X2 - X1))
+            return smaller[1] + (distance - smaller[0]) * ((larger[1]-smaller[1])/(larger[0]-smaller[0]));
```

Comment:
this technically isn't wrong because multiplication is commutative but it makes so much more sense if you do Y1 + (Y2 - Y1) * ((X-X1)/(X2-X1))